dipy/io/vtk.py

```diff
@@ -146,15 +147,19 @@ def save_vtk_streamlines(streamlines, filename,
     polydata.SetPoints(vtk_points)
     polydata.SetLines(vtk_lines)
 
-    writer = vtk.vtkPolyDataWriter()
-    writer.SetFileName(filename)
-    writer = utils.set_input(writer, polydata)
+    file_extension = os.path.splitext(filename)
+    if file_extension not in [".vtk", ".fib"]:
```

Comment:
You could add all the file extensions supported in the `save_polydata` function.